provider/offline_test.go

```diff
@@ -2310,12 +2366,20 @@ func testChainTransform(t *testing.T, store OfflineStore) {
 			t.Fatalf("Could not write value: %v: %v", err, value)
 		}
 	}
+
+	configQuery := ""
+	if store.Type() == "REDSHIFT_OFFLINE" {
+		configQuery = fmt.Sprintf("SELECT entity, \"int\", flt, str FROM %s", sanitize(table.GetName()))
```

Comment:
whats this needed for?

Reply:
Thats bad planning on my part. You can just modify the column name so its valid

Reply:
will make that change with the BigQuery PR change if you don't mind?

Reply:
Done! I thought it would be more complex change and wanted to close the PR.